src/abstractdataframe/reshape.jl

```diff
@@ -367,6 +389,10 @@ struct RepeatedVector{T} <: AbstractVector{T}
     outer::Int
 end
 
+Base.parent(v::RepeatedVector) = v.parent
+DataAPI.levels(v::RepeatedVector) = levels(parent(v))
+CategoricalArrays.isordered(v::RepeatedVector) = isordered(parent(v))
```

Comment:
```suggestion
CategoricalArrays.isordered(v::RepeatedVector{<:Union{CategoricalValue,CategoricalString}) = isordered(parent(v))
```

Reply:
Strictly speaking it is possible that some type defines `isordered` and does not expose the eltype.

But if you think we should add it it should be `AbstractVector{<:Union{CategoricalValue, CategoricalString, Missing}}` I think.

Apart from the simple case of `misssing` in the vector also something like this is true:
```
julia> eltype(categorical([1,2,"a"]))
Union{Missing, CategoricalValue{Any,UInt32}}
```
(have you intended this? - I guess yes, but want to double check)

Reply:
Ah yes. But I don't think we need to care about other types, for now at least `isordered` is only for CategoricalArrays.

Reply:
ok - restricted.